apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/responses/components/ResponseCardModal.tsx

```diff
@@ -108,7 +112,7 @@ export const ResponseCardModal = ({
             isViewer={isViewer}
             updateResponse={updateResponse}
             deleteResponses={deleteResponses}
-            setSelectedResponse={setSelectedResponse}
+            setSelectedResponse={setSelectedResponseForSingleResponseCard}
```

Comment:
I didn't replace setSelectedResponse with setSelectedResponseId because I think it could reduce the reusability of this component.
So i implemented a function `setSelectedResponseForSingleResponseCard`
I don't think this approach is the best.

Currently, setSelectedResponse is only used for resetting, so it might be a good idea to replace SelectedResponse with a prop called resetSelectedResponse

 I'd appreciate your opinion.